09-struct/conflation/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+)
+
+// START STRUCT OMIT
+type Circle struct {
+	X, Y, Radius int
+}
+// END STRUCT OMIT
+
+func main() { 
+// START BASIC OMIT
+	feet := 5.68
+	meters := 1.73
+	fmt.Println(feet + meters) // Is the result meaningful?
+
+	type Feet float64
+	type Meters float64
+	mile := Feet(5280)
+	km := Meters(1000)
+	// fmt.Println(mile + km) // type mismatch
```

Comment:
Yah, I'm not sure if we'll set up tests to ensure all the talk compiles. If we did this would be the best approach.

If not, I think uncommenting the line so as to easily show the error makes sense.